features/diffing.feature

```diff
@@ -25,13 +25,60 @@ Feature: diffing
     Then the output should contain:
       """
              Diff:
-             @@ -1,4 +1,4 @@
+             @@ -1,5 +1,5 @@
               this is the
              -  expected
              +  actual
                   string
       """
 
+  Scenario: diff for a multiline string with trailing newline
+    Given a file named "example_spec.rb" with:
+      """ruby
+      RSpec.describe "a multiline string" do
+        it "is like another string" do
+          expected = "Chewy\nbagel."
+          actual = "Chewy\nbagel.\n"
+          expect(actual).to eq(expected)
+        end
+      end
+      """
+    When I run `rspec example_spec.rb`
+    Then the output should contain:
+      """
+             Diff:
+             @@ -1,3 +1,4 @@
+              Chewy
+              bagel.
+             +
```

Comment:
Just for the records, [google-diff-match-patch](https://code.google.com/p/google-diff-match-patch/) represents trailing newlines no different than other newlines. 

Are you sure you want all of the code involved with putting in "\ No newline at end of input" in?  I'm totally for it, just checking one more time :)